spec/index.html

```diff
@@ -7882,23 +7882,16 @@ <h2>Definition of SPARQL</h2>
         abstract query is then evaluated on an RDF dataset.</p>
       <section id="initDefinitions">
         <h3>Initial Definitions</h3>
-        <section id="sparqlDataset">
-          <h4>RDF Dataset</h4>
-          <div class="defn">
-            <p><b>Definition: <span id="defn_RDFDataset">RDF Dataset</span></b></p>
-            <p>An RDF dataset is a set:<br>
-              { G, (&lt;u<sub>1</sub>&gt;, G<sub>1</sub>), (&lt;u<sub>2</sub>&gt;, G<sub>2</sub>), . .
-              . (&lt;u<sub>n</sub>&gt;, G<sub>n</sub>) }<br>
-              where G and each G<sub>i</sub> are graphs, and each &lt;u<sub>i</sub>&gt; is an IRI. Each
-              &lt;u<sub>i</sub>&gt; is distinct.</p>
-            <p>G is called the default graph. (&lt;u<sub>i</sub>&gt;, G<sub>i</sub>) are called named
-              graphs.</p>
-          </div>
+        <section id="activeGraph">
+          <h4>Active Graph</h4>
           <div class="defn">
             <p><b>Definition: <span id="defn_ActiveGraph">Active Graph</span></b></p>
             <p>The <b>active graph</b> is the graph from the dataset used for basic graph pattern
               matching.</p>
           </div>
+        </section>
+        <section id="rdfDatasetMerge">
```

Comment:
With graph [merging](https://www.w3.org/TR/rdf12-semantics/#dfn-merge) defined in RDF Semantics, perhaps it would make sense for RDFS to define Dataset Merging, and reference that, instead.

Reply:
Created a dedicated issue for this: https://github.com/w3c/sparql-query/issues/155